coss/home/models.py

```diff
@@ -107,3 +108,23 @@ class Testimonial(models.Model):
 
     def __str__(self):
         return self.person
+
+
+@register_snippet
+class FooterLink(models.Model):
+    """Model for dynamic Footer links."""
+
+    OPTIONS = ['Discourse', 'Github', 'Facebook', 'Twitter', 'Code of Conduct']
```

Comment:
Apart from this list we should allow content creators to create an new entry (link and icon).

Reply:
I'd argue that this is outside the scope of this PR. We need to discuss this idea more thoroughly, since this will allow content creators to build a footer that is completely outside of our current style theme.